js/output/pjs/pjs-ast-transforms.js

```diff
@@ -199,7 +199,7 @@ ASTTransforms.rewriteContextVariables = function(envName, context) {
                                         decl.init
                                     )
                                 ));
-                        } else {
+                            } else {
```

Comment:
Can you revert this this line?  The closing brace should line up with the `if` on line 189.